Remotion/Core/Development.Analyzers/Core.Development.Analyzers.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <LangVersion>9</LangVersion>
+    <Nullable>enable</Nullable>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DevelopmentDependency>true</DevelopmentDependency>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AssemblyName>Remotion.Core.Development.Analyzers</AssemblyName>
+    <AssemblyTitle>Remotion Core Analyzers</AssemblyTitle>
+    <Description>Analyzer ensuring best practices when using the Remotion.Web assemblies.</Description>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.8.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="$(OutputPath)\netstandard2.0\$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+  </ItemGroup>
```

Comment:
We are probably going to merge after #497 so we should do the same changes here. We want the analyzer project to not emit a nuget package, instead we want the Core project to embed the analyzer assembly. See the PR for the required changes